cmake/configs/default.cmake

```diff
@@ -60,6 +60,12 @@ hunter_config(Igloo VERSION 1.1.1-hunter)
 hunter_config(IntSizeof VERSION 1.0.0)
 hunter_config(Jpeg VERSION 9b-p1)
 hunter_config(JsonSpirit VERSION 0.0.4-hunter)
+if(MSVC_VERSION LESS 1600)
+    # for VS10 - version without support C++11
+  hunter_config(jsoncpp VERSION 0.7.0)
```

Comment:
Should be all green now :
the name of the imported library generated by find_package(jsoncpp... is not the same with 0.7.0 and 1.7.7. So I added a if(.. statement in the jsoncpp test